tracing-attributes/src/lib.rs

```diff
@@ -143,6 +145,12 @@ pub fn instrument(args: TokenStream, item: TokenStream) -> TokenStream {
     // function name
     let ident_str = ident.to_string();
 
+    // Pull out the arguments-to-be-skipped first, so we can filter results below.
+    let skips = match skips(&args) {
+        Ok(skips) => skips,
+        Err(err) => return quote!(#err).into(),
```

Comment:
Nit: I think this could just be
```suggestion
        Err(err) => err.into(),
```

Reply:
Hmm, had I tried that first and ran into a type error.

Reply:
oh, hm, nevermind then! let's just leave this as-is.